microsoft-365/enterprise/cross-tenant-mailbox-migration.md

```diff
@@ -513,22 +519,56 @@ TestUser_8@SourceCompany.onmicrosoft.com         {FullAccess}
 ```
  
 > [!Note]
 > Cross-tenant mailbox and calendar permissions are NOT supported. You must organize principals and delegates into consolidated move batches so that these connected mailboxes are transitioned at the same time from the source tenant. 
- 
-## Known issues 
+
+**What X500 proxy should be added to the target MailUser proxy addresses to enable migration?**  
+
+The cross-tenant mailbox migration requires that the LegacyExchangeDN value of the source mailbox object to be stamped as an x500 email address on the target MailUser object.  
+
+Example:  
+```powershell
+LegacyExchangeDN value on source mailbox is:  
+/o=First Organization/ou=Exchange Administrative Group(FYDIBOHF23SPDLT)/cn=Recipients/cn=d11ec1a2cacd4f81858c81907273f1f9Lara  
+
+so the x500 email address to be added to target MailUser object would be:  
+x500:/o=First Organization/ou=Exchange Administrative Group (FYDIBOHF23SPDLT)/cn=Recipients/cn=d11ec1a2cacd4f81858c81907273f1f9-Lara  
+```
+
+> [!Note]  
+> In addition to this X500 proxy, you will need to copy all X500 proxies from the mailbox in the source to the mailbox in the target.  
+
+**Is Azure Key Vault required and when are transactions made?**  
+
+Yes, an Azure subscription is required to use Key Vault to store the certificate to authorize migration. Unlike onboarding migrations which use username & password to authenticate to the source, cross-tenant mailbox migrations use OAuth and this certificate as the secret/credential. Access to the Key Vault must be maintained throughout all mailbox migrations as it is accessed once at the beginning and once end of migration, as well as once every 24 hours during incremental sync times. You can review AKV costing details [here]( https://azure.microsoft.com/en-us/pricing/details/key-vault/).  
+
+**Do you have any recommendations for batches?**  
+
+Do not exceed 2000 mailboxes per batch. We strongly recommend submitting batches two weeks prior to the cut-over date as there is no impact to the end users during sync. If you need guidance for mailboxes quantities over 50,000 you can reachout to the Engineering Feedback Distribution List at crosstenantmigrationpreview@service.microsoft.com.
+
+**What if I use Service encryption with Customer Key?**
+
+The mailbox will be decrypted prior to moving. Ensure Customer Key is configured in the target tenant if it is still required. See [here](https://docs.microsoft.com/microsoft-365/compliance/customer-key-overview) for more information.  
+
+**What is the estimated migration time?**
+
+To help you plan your migration, the table present [here](https://docs.microsoft.com/exchange/mailbox-migration/office-365-migration-best-practices#estimated-migration-times) shows the guidelines about when to expect bulk mailbox migrations or individual migrations to complete. These estimates are based on a data analysis of previous customer migrations. Because every environment is unique, your exact migration velocity may vary.  
```

Comment:
```suggestion
To help you plan your migration, this [estimated migration times](https://docs.microsoft.com/exchange/mailbox-migration/office-365-migration-best-practices#estimated-migration-times) table gives guidelines about when to expect bulk mailbox migrations or individual migrations to complete. These estimates are based on data analysis of previous customer migrations. Because every environment is unique, your exact migration velocity may vary.  
```